services/api/mw2pdf/INSTALL.md

```diff
@@ -1,8 +1,12 @@
 # Installing
 
 This script requires node.js to be installed.
-In order to run the script, you will first need to build the dependencies using `npm install`
+In order to run the script, you will first need to build the dependencies using `yarn install`
 
-## Running the script
+# Building
 
-You can run the script using `node index.js`
\ No newline at end of file
+`yarn build`
```

Comment:
Not for this PR, but maybe a good immediate followup task: now that building is a thing... we should probably add something to devenv that automatically builds this package whenever files are changed.

Could you make an issue in devenv to that effect?

Reply:
I'm sorry but I don't quite understand what you are asking here.

Reply:
Sorry about that!

So basically in the current setup we have:

1) `SimpleBook` mounted into devenv
2) Python triggering a node process that runs `node index.js {book params}`

This means in the current setup if I were to edit `index.js` that change would actually be reflected on the development environment processes without me taking any extra steps.  If I triggered a new book, that new book would be processed using whatever logic I have.

Now that we're doing typescript / webpack I'm under the impression that flow won't be quite as seemless (by default).  Specifically: the typescript has to be built into javascript after each change in order for the change to be reflected in the live process.

I'm just suggesting we should:

1) add something like `nodemon` or whatever, which will watch the source directory and trigger `yarn build` any time a source file is changed.  This could be wrapped in a `yarn watch` command, or not.
2) add a `supervisor` task to devenv which keeps that watch-and-build process running.

The goal is that we don't want developers to have to manually run `yarn build` every time they make a change and want to test it locally.

Hopefully this makes more sense but let me know if not.

Reply:
> Sorry about that!

Not your fault!

> 
> So basically in the current setup we have:
> 
>     1. `SimpleBook` mounted into devenv
> 
>     2. Python triggering a node process that runs `node index.js {book params}`
> 
> 
> This means in the current setup if I were to edit `index.js` that change would actually be reflected on the development environment processes without me taking any extra steps. If I triggered a new book, that new book would be processed using whatever logic I have.
> 
> Now that we're doing typescript / webpack I'm under the impression that flow won't be quite as seemless (by default). Specifically: the typescript has to be built into javascript after each change in order for the change to be reflected in the live process.

You are absolutely correct!

> 
> I'm just suggesting we should:
> 
>     1. add something like `nodemon` or whatever, which will watch the source directory and trigger `yarn build` any time a source file is changed.  This could be wrapped in a `yarn watch` command, or not.
> 
>     2. add a `supervisor` task to devenv which keeps that watch-and-build process running.
> 
> 

This sounds like a good plan!

> The goal is that we don't want developers to have to manually run `yarn build` every time they make a change and want to test it locally.

Agreed.

> 
> Hopefully this makes more sense but let me know if not.

This makes much more sense! 

I will open an issue in devenv to do this work!

Will

Reply:
This is now done via https://github.com/OpenTechStrategies/torque-devenv/issues/19. I will let @slifty resolve this when they think that it is resolved.